pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -852,7 +852,7 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       await this.writeFile(`/etc/moproxy/proxy.ini`, '; no proxy defined');
     }
 
-    await this.modifyConf('moproxy', { MOPROXY_NOPROXY: proxy.noproxy.join(',') });
+    await this.modifyConf('moproxy', { MOPROXY_NOPROXY: proxy.noproxy.filter(ip => ip.trim()).join(',') });
```

Comment:
Pretty sure you never want `a.filter(x => x.trim())`, but instead `a.map(x => x.trim()).filter(x => x)` instead (since currently you're not keeping the result of the trim).